src/reporting/reporting_app/settings/base.py

```diff
@@ -1,17 +1,41 @@
 # Django settings for reporting_app project.
-import os
+import ldap
+from django_auth_ldap.config import LDAPSearch, PosixGroupType
+from django.core.exceptions import ImproperlyConfigured
+from os import environ
+from pathlib import Path
 import django
 
 # The DB settings are defined in the workflow manager
 from workflow.database.settings import DATABASES
 
+
+def validate_ldap_settings(server_uri, user_dn_template):
```

Comment:
Since the two variables we are injecting are `LDAP_SERVER_URI` and `LDAP_DOMAIN_COMPONENT` we should check that those parts are set correctly. The `server_uri` is fine, but the `user_dn_template` will always be non-empty. Instead, it should be checked to see if it is longer than `uid=%(user)s,ou=users,` which is the part filled in by`settings/base.py`. I'm suspicious that checking that the string is longer than the filled in part is enough.